src/main/java/math/Fraction.java

```diff
@@ -192,8 +195,7 @@ public Fraction reciprocal() {
      * @return the absolute value of this fraction
      */
     public Fraction abs() {
-        // TODO (X0Q03E)
-        return null;
+        return new Fraction(Math.abs(numerator),Math.abs(denominator));
```

Comment:
Note that the denominator is always positive, thus,  `return new Fraction(Math.abs(numerator), denominator)` also does the job.